torchaudio/compliance/kaldi.py

```diff
@@ -195,6 +175,59 @@ def spectrogram(
     if not raw_energy:
         signal_log_energy = _get_log_energy(strided_input, EPSILON, energy_floor)  # size (m)
 
+    return strided_input, signal_log_energy
+
+
+def spectrogram(
```

Comment:
Do you think this can easily be JIT-able?

Reply:
I guess we'll see in next PR